src/components/Navbar.js

```diff
@@ -0,0 +1,18 @@
+const Navbar = () => {
+  const links = [
+    { name: 'Books', path: '/' },
+    { name: 'Categories', path: 'categories' },
+  ];
+  return (
+    <nav className="navbar">
+      <h1>Bookstore CMS</h1>
+      <ul className="nav-links">
+        {links.map((link) => (
+          <li key={link.name}><a href={link.path}>{link.name}</a></li>
```

Comment:
- [ ] in order to make you project an SPA kindly use React Router Link instead of anchor tag to navigate to your pages

Reply:
Done!